### PR TITLE
PullFromSourceParallel small bug fixes

### PR DESCRIPTION
**Pull request by DavisPuciriuss**
Fixed issue with queues array not being filled correctly based on scout.chunk.handlers.
Added an option to change the indexing queue name for parallel import.

### CODE REVIEW

File: src/Jobs/Stages/PullFromSourceParallel.php [hunk 1]
```diff
@@ -19,6 +19,11 @@ final class PullFromSourceParallel implements StageInterface
      */
     const DEFAULT_HANDLER_COUNT = 1;
 
+    /**
+     * @var string
+     */
+    const DEFAULT_QUEUE_NAME = 'indexing-queue';
```
Comment (matchish) [inline src/Jobs/Stages/PullFromSourceParallel.php:25]: Better if less generic to prevent conflicts.

Comment (DavisPuciriuss) [reply src/Jobs/Stages/PullFromSourceParallel.php]: changed to 'elasticsearch-parallel'

**Comment by matchish** [comment]
Update changelog for last version https://github.com/matchish/laravel-scout-elasticsearch/blob/8.x/CHANGELOG.md#800-alpha2---2024-

lets release them together. and fix date(it's not complete)

**Comment by DavisPuciriuss** [comment]
Updated CHANGELOG.md and synced with 8.x branch.

**Comment by matchish** [comment]
Thank you, @DavisPuciriuss, for your valuable contributions!

**Comment by matchish** [comment]
released in 8.0.0-alpha.2

**Pull request merged**